### PR TITLE
Fix typo in PSAP section for NFD operator

### PR DESCRIPTION
**Pull request by GroceryBoyJr**
Version(s): 4.13+

Issue:  https://issues.redhat.com/browse/OSDOCS-11669

Link to docs preview: https://79781--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hardware_enablement/psap-node-feature-discovery-operator.html

QE review:
- [*] QE has approved this change.

Additional information: Principal and Sr SW Engs approved. ContentX_24 improvement.

### CODE REVIEW

**Comment by GroceryBoyJr** [comment]
@liqcui your qe approval desired (or your designate, please)

File: modules/creating-nfd-cr-cli.adoc [hunk 1]
```diff
@@ -8,6 +8,11 @@
 
 As a cluster administrator, you can create a `NodeFeatureDiscovery` CR instance by using the {oc-first}.
 
+[NOTE]
+====
+The `spec.operand.image` setting now requires a `-rhel9` image to be defined for use with the 4.16 release of OpenShift Container Platform. The following example shows the use of `-rhel9` to acquire the correct image.
```
[the remark this answers is not in the document]
Comment (GroceryBoyJr) [reply modules/creating-nfd-cr-cli.adoc]: This is fixed.

**Review by dry923 — APPROVED** [review]
LGTM

**Review by gryf — COMMENTED** [review]
/lgtm

**Review by gwynnemonahan — COMMENTED** [review]
Hey @GroceryBoyJr ,

LGTM.

File: modules/creating-nfd-cr-cli.adoc [hunk 2]
```diff
@@ -8,6 +8,11 @@
 
 As a cluster administrator, you can create a `NodeFeatureDiscovery` CR instance by using the {oc-first}.
 
+[NOTE]
+====
+The `spec.operand.image` setting now requires a `-rhel9` image to be defined for use with the 4.16 release of {product-title}. The following example shows the use of `-rhel9` to acquire the correct image.
```
Comment (jab-rh) [inline modules/creating-nfd-cr-cli.adoc:13]: @GroceryBoyJr, is this for only the 4.16 release? What about if it is the 4.17 release?

Comment (GroceryBoyJr) [reply modules/creating-nfd-cr-cli.adoc:13]: Actually, good point, @jab-rh . I discovered this week it's for 4.13+! So I'll edit to support that finding.....

Comment (GroceryBoyJr) [reply modules/creating-nfd-cr-cli.adoc:13]: So...while this statement is true, it needs to be made....true-er!  :-)

Comment (GroceryBoyJr) [reply modules/creating-nfd-cr-cli.adoc]: Good point, @jab-rh . I have corrected this. Also, I removed the second sentence from the NOTE box, which is referring to the example. No need for that to be in the NOTE box.

**Comment by ShaunaDiaz** [comment]
One sour cherry needs special treatment, @GroceryBoyJr https://github.com/openshift/openshift-docs/pull/79781#issuecomment-2289482055. LMK when ready and I can merge it up for you.

**Pull request merged**